dmasif_embedding_utils.py

```diff
@@ -0,0 +1,165 @@
+import os
+import sys
+sys.path.append("/mnt/data/MaSIF_colab") 
```

Comment:
this seems to hardcode some paths that might not be available in every computer. Is there another way to do this, e.g. passing paths by a command line flag?

Reply:
You're right! I don't even need to do these commands, I must have left them from the notebook. But I removed the unecessary hardcoded path from the main script too. I don't need it! Thank you :)